packages/common/src/store/account/sagas.ts

```diff
@@ -1,26 +1,31 @@
 import { call, put, select, takeLatest } from 'typed-redux-saga'
 
-import { getContext } from '~/store/effects'
+import { Id } from '~/models'
 import { UPLOAD_TRACKS_SUCCEEDED } from '~/store/upload/actions'
 
+import { getSDK } from '../sdkUtils'
+
 import { getUserId, getUserHandle } from './selectors'
 import { fetchAccountSucceeded, fetchHasTracks, setHasTracks } from './slice'
 
 function* handleFetchTrackCount() {
   const currentUserId = yield* select(getUserId)
   const handle = yield* select(getUserHandle)
-  const apiClient = yield* getContext('apiClient')
+  const sdk = yield* getSDK()
 
   if (!currentUserId || !handle) return
 
   try {
-    const tracks = yield* call([apiClient, apiClient.getUserTracksByHandle], {
-      handle,
-      currentUserId,
-      getUnlisted: true
-    })
-
-    yield* put(setHasTracks(tracks.length > 0))
+    const { data = [] } = yield* call(
+      [sdk.full.users, sdk.full.users.getTracksByUserHandle],
+      {
+        handle,
+        userId: Id.parse(currentUserId),
+        limit: 1
```

Comment:
This is new. Noticed that we aren't actually using the response here, just checking if the list is > 0. So we don't need to fetch more than one track.

Reply:
Great change!